shared/Microsoft.AspNetCore.Razor.TagHelpers.Testing.Sources/CaseSensitiveBoundAttributeComparer.cs

```diff
@@ -35,11 +34,7 @@ public bool Equals(TagHelperAttribute attributeX, TagHelperAttribute attributeY)
 
         public int GetHashCode(TagHelperAttribute attribute)
         {
-            var hashCodeCombiner = HashCodeCombiner.Start();
-            hashCodeCombiner.Add(attribute.GetHashCode());
-            hashCodeCombiner.Add(attribute.Name, StringComparer.Ordinal);
-
-            return hashCodeCombiner.CombinedHash;
```

Comment:
Adding a source project to tests makes things get nasty. This isn't important so I just simplified it.